### PR TITLE
Add extra documentation for old plugin users

### PR DESCRIPTION
**Pull request by andrewvc**

### CODE REVIEW

**Comment by andrewvc** [comment]
@ph let me know if this works for you!

File: lib/logstash/outputs/sns.rb [hunk 1]
```diff
@@ -24,6 +24,12 @@
 #  with the @message truncated so that the length of the JSON fits in
 #  `MAX_MESSAGE_SIZE_IN_BYTES`.
 #
+# ## Upgrading to 2.0.0
+#
+# This plugin used to have a `format` option for controlling the encoding of messages prior to being sent to SNS.
+  # This plugin now uses the logstash standard https://www.elastic.co/guide/en/logstash/current/configuration.html#codec[codec] option for encoding instead.
```
Comment (ph) [inline lib/logstash/outputs/sns.rb]: remove the space at the start of the line

**Comment by andrewvc** [comment]
@ph removed the line issue. Apologies

File: lib/logstash/outputs/sns.rb [hunk 2]
```diff
@@ -24,6 +24,12 @@
...
+# For example, rather than using `format => "plain"` use `codec => "s3_plain"`.
```
Comment (ph) [inline lib/logstash/outputs/sns.rb]: Probably mention that If you are upgrading from v1 to v2 to use that codec if you want the same format.

**Comment by ph** [comment]
@andrewvc Made another comments that was hidden with the new push :)

**Comment by ph** [comment]
You will need to bump the version too..

**Comment by andrewvc** [comment]
@ph bumped that as well. How's it look?

**Comment by ph** [comment]
look great rebase and release :)

**Comment by andrewvc** [comment]
@ph this is rebased unless I'm missing something?

**Comment by ph** [comment]
Ah yes it is, make sense to have 3 commits.

**Comment by ph** [comment]
Update `changelog.md` I always forget it.. still new to me.

**Pull request closed without merging**